### DISCUSSION TITLE
I really need help, i need to reset my phone, but I have either 773 tabs or 773 sessions saved in kiwiBrowser(NOT bookmarks) I only have 5 bookmarks.

### DISCUSSION THREAD

**Discussion by Username23453** [floccusaddon/floccus#1399] [Q&A] [answered]
And I need to transfer the 773 to my desktop somehow, so i can review them there. I know about floccus, but its asking whether I want to use WebDAV, i dont know how to use that, & also its asking if i want to use my google drive. I was about to try this. But, i'm concerned on how this would work. I have desktop browser profiles connected to my google drive account, with saved history,sessions, bookmarks, & I'm concerned floccus might delete my kiwi stuff & move my desktop browser profile data to my kiwi browser. I need some help, urgently, a walkthrough or something. My phone is lagging extremely & serious other problems & I urgently need to reset the OS.
I've also tried "Kiwi Sync" extension, but it didnt work at all.
-Thank you.

**Comment by marcelklehr** [comment]
Hello :wave: 

> its asking whether I want to use WebDAV, i dont know how to use that

Then don't use it.

>  I have desktop browser profiles connected to my google drive account

To your drive account? Really? With floccus? Or are you simply syncing your browser via your google account?

In case of the latter, you should be fine to sync your tabs to google drive.

> I'm concerned floccus might delete my kiwi stuff

Floccus allows you to specify to only sync stuff up without changing the local data in any way.

To rescue your tabs you should be able to 

1. Install floccus on Kiwi
2. Set up a sync account to google drive (make sure to select that you want to sync *tabs* no bookmarks folder)
3. Trigger a push sync
4. Now your tabs should be in a file in google drive

**Reply by Username23453** [reply]
Thanks!  When I install Floccus in kiwi as an extension, i get to the ""Login with google" button with floccus, but when i click it, it doesnt do anything, gets stuck. :( I've tried to click it many times.

**Reply by marcelklehr** [reply]
Ah, they still haven't fixed that. Damn. Then you'll need to go a different route:

On some desktop browser:

1. Install floccus
2. Setup the sync account you would have setup on kiwi
3. Export your accounts to a json file
4. Transfer the json file to your phone
5. import the json file into floccus on kiwi

**Reply by Username23453** [reply]
@marcelklehr , thanks!  So i did all that, its not syncing my 695 tabs from kiwi, when i tell it to sync, the json file doesnt change at all. See [screenshot](https://i2.paste.pics/OGU87.png).   Does it only sync bookmarks? i thought it synced tabs or saved sessions too?

**Reply by marcelklehr** [reply]
> the json file doesnt change at all

The JSON file is not supposed to get syned. You should look for a file called bookmarks.xbel in your google drive. That's where your tabs are in now.

**Reply by marcelklehr** [reply, marked as answer]
If you like you can also join the floccus matrix room: https://matrix.to/#/#marcelklehr_floccus:gitter.im I might be able to help you quicker there :)

**Reply by Username23453** [reply]
@marcelklehr ,thanks! i sent you a message on matrix :)
